packages/admin/src/targetGroups/TargetGroupsGrid.tsx

```diff
@@ -50,6 +50,7 @@ const targetGroupsFragment = gql`
         totalSubscribers
         totalContactsBlocked
         isMainList
+        isTestList
```

Comment:
This is not used in the frontend, can this be removed

Reply:
will be false for every target group anyways since it gets filtered in the api

Reply:
51c898ce3c997e11ef38c54df1ba856b202a056c